prep-javascript-dom-manipulation/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>JavaScript DOM Manipulation</title>
+  <style>
+    #container {
+      border: 2px solid green;
+    }
+  </style>
+</head>
+<body>
+  <div id="container">
+    <p class="first-paragraph">
+      I really like coding.
+    </p>
+    <p class="second-paragraph">
+
+    </p>
+    <p class="third-paragraph">
+
+    </p>
+  </div>
+  <script>
+    // Your code here;
+const $secondParagraph = document.getElementsByClassName('second-paragraph');
+console.log($secondParagraph);
+
+document.getElementsByClassName('second-paragraph').innerHTML = 'I also really like to watch television and eating pancakes.';
+
+const $thirdParagraph = document.getElementsByClassName('third-paragraph');
+console.log($thirdParagraph);
+
+document.getElementsByClassName('third-paragraph').innerHTML = 'I hope one day to meet people that also like coding, television, and pancakes.';
```

Comment:
In this one we can use `document.querySelector()` to grab references to the paragraph elements.

```js
var $secondParagraph = document.querySelector('.second-paragraph');
var $thirdParagraph = document.querySelector('.third-paragraph');
```

Once we have variables for those elements, we can assign strings to their `textContent` properties.

```js
$secondParagraph.textContent = 'I also really like to watch television and eating pancakes.';
$thirdParagraph.textContent = 'I hope one day to meet people that also like coding, television, and pancakes.';
```